h2o-py/h2o/explanation/_explain.py

```diff
@@ -2530,7 +2530,7 @@ def learning_curve_plot(
 
     timestep = allowed_timesteps[0]
 
-    if "deviance" == metric and model.algo in ["glm", "gam"] and not model.actual_params.get("HGLM", False):
+    if "deviance" == metric and model.algo in ["glm", "gam"] and not model.actual_params.get("HGLM", False) and "deviance_train" in scoring_history.col_header:
```

Comment:
I kind of hoped that this error was caused by GLM starting to behave like the other algos - having `training_{metric}` and `validation_{metric}`. Does (H)GLM/GAM still use `deviance_train`?

Reply:
Maybe @wendycwong would know? Also this issue is most likely present in R client as well.

Reply:
also, not related with the discussion above, but we have 4 conditions here, so for improved readability don't hesitate to:
```python
if (cond
    and cond
    and cond
    ...
    and cond):
```